operations/app/terraform/vars/skylight/variables.tf

```diff
@@ -0,0 +1,124 @@
+# TODO: not currently used
+variable "tf_secrets_vault" {
+  default = "piskylight-tf-secrets"
+}
+
+## Set basic variables
+variable "terraform_object_id" {
+  type        = string
+  description = "Object id of user running TF"
+  # NOTE: set to object ID of CT-DMZ-PRIME-INGESTION-TST-AZ-Contributor
+  default = "9ff42a69-beb8-4b4a-9406-e7fbcbf847ee"
+}
+
+variable "app_subnet_name" {
+  default = "app"
+}
+
+variable "cdc_vnet_name" {
+  default = "prime-ingestion-skylight-VNET"
+}
+
+variable "environment" {
+  default = "skylight"
+}
+
+variable "https_cert_names" {
+  default = []
+}
+
+variable "location" {
+  default = "eastus"
+}
+
+variable "resource_group_name" {
+  default = "prime-ingestion-skylight"
+}
+
+variable "resource_prefix" {
+  default = "piskylight"
+}
+
+variable "rsa_key_2048" {
+  default = null
+}
+
+variable "rsa_key_4096" {
+  default = null
+}
+
+variable "service_subnet_name" {
+  default = "service"
+}
+
+variable "route_table_name" {
+  default = "prime-ingestion-skylight-RT"
+}
+
+variable "route_table_resource_group_name" {
+  default = "prime-ingestion-skylight"
+}
+
+variable "publish_functions" {
+  type    = bool
+  default = true
+}
+
+variable "aad_object_keyvault_admin" {
```

Comment:
I'm guessing this correctly says `aad` instead of `add`?

Reply:
Yep, aad should stand for Azure Active Directory here